lib/src/functions/color.dart

```diff
@@ -776,3 +775,8 @@ SassColor _transparentize(List<Value> arguments) {
 
 /// Like [fuzzyRound], but returns `null` if [number] is `null`.
 int _fuzzyRoundOrNull(num number) => number == null ? null : fuzzyRound(number);
+
+/// Like [new BuiltInCallable.function], but always sets the URL to `sass:math`.
```

Comment:
URL in comment should be `sass:color`. This got copied to all the other files under `lib/src/function`. All of them need updating... except for math.dart.

Reply:
Done.